libs/wingsdk/src/target-sim/api.inflight.ts

```diff
@@ -131,8 +130,13 @@ export class Api
 
   public async cleanup(): Promise<void> {
     this.addTrace(`Closing server on ${this.url}`);
-    this.server?.close();
-    this.server?.closeAllConnections();
```

Comment:
@Chriscbr fyi (really hard to test but I ran into this)

Reply:
More stability! 🙌  nice fix